### ISSUE TITLE
2.7 and above non-descriptive build error

### ISSUE THREAD

**Issue by ScJa** [Triple-T/gradle-play-publisher#778]
Hey, 

I just spent some time migrating a project to use GPP and it took me quite a while to get everything working.

Reason for this is that using the latest version of GPP (2.7.1) fails to sync/build with a non-descriptive error: 
```
A problem occurred evaluating project ':app'.
> com/android/Version
```
The stacktrace does not give much more background.

After more debugging I found out that 2.7.1 only works with beta versions of AGP. 
Would be great to have a descriptive error message if there is a dependency missing. 
I'm not super familiar with ecosystem but is it normal that public releases depend on beta versions of other dependencies?

You can reproduce this by using 
```
        classpath 'com.android.tools.build:gradle:3.5.3'
```
as a dependency

Thanks!

**Comment by b95505017** [comment]
By the way AGP 3.6.0 stable is out.

**Comment by SUPERCILEX** [comment]
Sorry about that, I'll fix it in 2.7.2. 👍

> I'm not super familiar with ecosystem but is it normal that public releases depend on beta versions of other dependencies?

No, but I really hated supporting 3.5.x and made a bet Google would ship 3.6 stable this week. I got lucky! 😅

**Comment by ScJa** [comment]
Thanks for quick response :) We rely on this API at Google a lot.
